cmd/juju/status/status.go

```diff
@@ -401,29 +400,43 @@ func (c *statusCommand) runStatus(ctx *cmd.Context) error {
 	return nil
 }
 
-// clearScreen removes any character from the terminal
-// using ANSI scape characters.
-func clearScreen() {
-	fmt.Printf("\u001Bc")
-}
-
 func (c *statusCommand) Run(ctx *cmd.Context) error {
 	defer c.close()
 
 	if c.watch != 0 {
-		clearScreen()
-	}
+		//  Get list of all args, cut off '--watch' flag from 'juju status'
+		var jujuStatusArgsWithoutWatch []string
+		var watchValue string
+		for i := range os.Args {
+			if os.Args[i] == "--watch" {
+				watchValue = os.Args[i+1]
+				jujuStatusArgsWithoutWatch = append(os.Args[:i], os.Args[i+2:]...)
+				break
+			}
+		}
 
-	err := c.runStatus(ctx)
-	if err != nil || c.watch == 0 {
-		return err
-	}
+		// Create Viddy config
+		v := viper.New()
+		v.SetConfigType("toml")
+		v.SetConfigName("viddy")
+		v.AddConfigPath(xdg.ConfigHome)
```

Comment:
effectively this means we are embedding viddy as a CLI tool, and not just using it as a library.  (we're populating a $HOME path for it to read config from)
This has quite a bit of implication. If we do this for status, we'll have to reproduce this for any other command where we want watch-like behavior. (and we certainly wouldn't want them to be different).
If we do want to embed viddy, I still think we don't want its code sprinkled around StatusCommand like this.

Reply:
That's reasonable,

What is the example of such a 'universal' flag for different commands? `--format`?
Maybe I can suggest viddy integration in the same manner...

Reply:
Now viddy config generation is a part of juju/viddy package